torchci/rockset/commons/__sql/weekly_force_merge_stats.sql

```diff
@@ -96,54 +103,43 @@ WITH
             ) AS force_merge_with_failures,
             CAST(time as DATE) as date
         FROM
-            force_merges_with_failed_checks
+            merges_identifying_force_merges
         ORDER BY
```

Comment:
Nit: Just notice this `ORDER BY` part, could it be removed to make the query a bit faster?  It looks like it's not needed here